### ISSUE TITLE
Autokey breaks dependencies in the latest KDE Neon.

### ISSUE THREAD

**Issue by nemonein** [autokey/autokey#557]
## Classification:

Bug

## Reproducibility:

Always

## Version

AutoKey version: 0.95.10

Used GUI (Gtk, Qt, or both): Qt

If the problem is known to be present in more than one version, please list all of those.

Installed via: PPA (nemonein)

Linux Distribution: KDE Neon

## Summary

In the latest update on KDE Neon(a few days ago), I've got these messages.

```
 
Starting pkgProblemResolver with broken count: 1
Starting 2 pkgProblemResolver with broken count: 1
Investigating (0) python3-pyqt5:amd64 < 5.14.1+dfsg-3+20.04+focal+build13 -> 5.15.4+dfsg-1+20.04+focal+unstable+build20 @ii umU Ib >
Broken python3-pyqt5:amd64 breaks on python3-pyqt5.qsci:amd64 < 2.11.2+dfsg-6 @ii mK > (< 2.11.5~)
  Considering python3-pyqt5.qsci:amd64 0 as a solution to python3-pyqt5:amd64 8
  Added python3-pyqt5.qsci:amd64 to the remove list
  Fixing python3-pyqt5:amd64 via remove of python3-pyqt5.qsci:amd64
Investigating (0) autokey-qt:amd64 < 0.95.10-0-0build1~ubuntu20.04 @ii mK Ib >
Broken autokey-qt:amd64 depends on python3-pyqt5.qsci:amd64 < 2.11.2+dfsg-6 @ii mR >
  Considering python3-pyqt5.qsci:amd64 0 as a solution to autokey-qt:amd64 0
  Removing autokey-qt:amd64 rather than change python3-pyqt5.qsci:amd64
```

Maybe these problems already are solved on 0.96?

**Comment by josephj11** [comment]
I don't think  any work on 0.96 touched on qscintilla other than allowing you to change the schema (theme∕/colors). What version, of `python3-pyqt5.qsci:amd64` is installed on your system? I don't see that in the traceback.

**Comment by nemonein** [comment]
I'm sorry. It certainly is the problem of KDE-Neon.
The version of ```python3-pyqt5.qsci``` is 2.11.2+dfsg-6.
The version of ```python3-pyqt5``` is 5.15.4+dfsg-1+20.04+focal+unstable+build20, but the one in Ubuntu 20.04 is 5.14.1.

There might be some issues between python3-pyqt5 and python3-pyqt5.qsci.
I can't install pyqt5.qsci.
```
sudo apt install python3-pyqt5.qsci                                                                    
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Starting pkgProblemResolver with broken count: 1
Starting 2 pkgProblemResolver with broken count: 1
Investigating (0) python3-pyqt5:amd64 < 5.15.4+dfsg-1+20.04+focal+unstable+build20 @ii mK Ib >
Broken python3-pyqt5:amd64 Breaks on python3-pyqt5.qsci:amd64 < none -> 2.11.2+dfsg-6 @un puN > (< 2.11.5~)
  Considering python3-pyqt5.qsci:amd64 9998 as a solution to python3-pyqt5:amd64 5
  Removing python3-pyqt5:amd64 rather than change python3-pyqt5.qsci:amd64
Investigating (0) ubuntu-release-upgrader-qt:amd64 < 1:20.04.33+p20.04+trelease+git20210428.1007 @ii mK Ib >
Broken ubuntu-release-upgrader-qt:amd64 Depends on python3-pyqt5:amd64 < 5.15.4+dfsg-1+20.04+focal+unstable+build20 @ii mR >
  Considering python3-pyqt5:amd64 5 as a solution to ubuntu-release-upgrader-qt:amd64 3
  Removing ubuntu-release-upgrader-qt:amd64 rather than change python3-pyqt5:amd64
Investigating (0) distro-release-notifier:amd64 < 0.0+p20.04+trelease+git20210518.0013-0 @ii mK Ib >
Broken distro-release-notifier:amd64 Depends on ubuntu-release-upgrader-qt:amd64 < 1:20.04.33+p20.04+trelease+git20210428.1007 @ii mR >
  Considering ubuntu-release-upgrader-qt:amd64 3 as a solution to distro-release-notifier:amd64 1
  Removing distro-release-notifier:amd64 rather than change ubuntu-release-upgrader-qt:amd64
Investigating (0) software-properties-qt:amd64 < 0.98.9.5 @ii mK Ib >
Broken software-properties-qt:amd64 Depends on python3-pyqt5:amd64 < 5.15.4+dfsg-1+20.04+focal+unstable+build20 @ii mR >
  Considering python3-pyqt5:amd64 5 as a solution to software-properties-qt:amd64 0
  Removing software-properties-qt:amd64 rather than change python3-pyqt5:amd64
Investigating (0) neon-desktop:amd64 < 4+p20.04+trelease+git20210525.0550 @ii mK NPb Ib >
Broken neon-desktop:amd64 Depends on distro-release-notifier:amd64 < 0.0+p20.04+trelease+git20210518.0013-0 @ii mR >
  Considering distro-release-notifier:amd64 1 as a solution to neon-desktop:amd64 0
  Removing neon-desktop:amd64 rather than change distro-release-notifier:amd64
Investigating (1) python3-pyqt5.qsci:amd64 < none -> 2.11.2+dfsg-6 @un puN Ib >
Broken python3-pyqt5.qsci:amd64 Depends on python3-pyqt5:amd64 < 5.15.4+dfsg-1+20.04+focal+unstable+build20 @ii mR >
  Considering python3-pyqt5:amd64 5 as a solution to python3-pyqt5.qsci:amd64 9998
  Added python3-pyqt5:amd64 to the remove list
  Fixing python3-pyqt5.qsci:amd64 via keep of python3-pyqt5:amd64
Investigating (1) python3-pyqt5:amd64 < 5.15.4+dfsg-1+20.04+focal+unstable+build20 @ii mK Ib >
Broken python3-pyqt5:amd64 Breaks on python3-pyqt5.qsci:amd64 < none -> 2.11.2+dfsg-6 @un puN > (< 2.11.5~)
  Considering python3-pyqt5.qsci:amd64 9998 as a solution to python3-pyqt5:amd64 5
  Removing python3-pyqt5:amd64 rather than change python3-pyqt5.qsci:amd64
Investigating (2) python3-pyqt5.qsci:amd64 < none -> 2.11.2+dfsg-6 @un puN Ib >
Broken python3-pyqt5.qsci:amd64 Depends on python3-pyqt5:amd64 < 5.15.4+dfsg-1+20.04+focal+unstable+build20 @ii mR >
  Considering python3-pyqt5:amd64 5 as a solution to python3-pyqt5.qsci:amd64 9998
  Added python3-pyqt5:amd64 to the remove list
  Fixing python3-pyqt5.qsci:amd64 via keep of python3-pyqt5:amd64
Investigating (2) python3-pyqt5:amd64 < 5.15.4+dfsg-1+20.04+focal+unstable+build20 @ii mK Ib >
Broken python3-pyqt5:amd64 Breaks on python3-pyqt5.qsci:amd64 < none -> 2.11.2+dfsg-6 @un puN > (< 2.11.5~)
  Considering python3-pyqt5.qsci:amd64 9998 as a solution to python3-pyqt5:amd64 9998
  Removing python3-pyqt5:amd64 rather than change python3-pyqt5.qsci:amd64
Investigating (3) python3-pyqt5.qsci:amd64 < none -> 2.11.2+dfsg-6 @un puN Ib >
Broken python3-pyqt5.qsci:amd64 Depends on python3-pyqt5:amd64 < 5.15.4+dfsg-1+20.04+focal+unstable+build20 @ii mR >
  Considering python3-pyqt5:amd64 9998 as a solution to python3-pyqt5.qsci:amd64 9998
  Considering python3-pyqt5:amd64 9998 as a solution to python3-pyqt5.qsci:amd64 9998
  Considering python3-pyqt5:amd64 9998 as a solution to python3-pyqt5.qsci:amd64 9998
  Considering python3-pyqt5:amd64 9998 as a solution to python3-pyqt5.qsci:amd64 9998
  Considering python3-pyqt5:amd64 9998 as a solution to python3-pyqt5.qsci:amd64 9998
Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 python3-pyqt5.qsci : Depends: python3-pyqt5 but it is not going to be installed
E: Unable to correct problems, you have held broken packages.
```

Because it's not autokey's responsibility, I close this issue.

**Comment by josephj11** [comment]
While it's probably not our problem *yet*, you might want to consider leaving this open until our devs get a chance to look at it. It might be something they'll have to deal with soon. IDK

**Comment by nemonein** [comment]
Ok, then.
Thanks.

**Comment by Vinzent03** [comment]
I can confirm that autkey was missing out of nowhere on my system, so I tried to reinstall it with `sudo apt-get install autokey-qt`, but that didn't work, because I can't install `python3-pyqt5.qsci` 
(I am on KDE neon 5.21.5)
Sad that I can't use autkey anymore for now.

**Comment by nemonein** [comment]
I reported this to the [KDE team](https://bugs.kde.org/show_bug.cgi?id=438211). I hope this would be fixed soon.

**Comment by blue-bullet** [comment]
> I reported this to the [KDE team](https://bugs.kde.org/show_bug.cgi?id=438211). I hope this would be fixed soon.

Thank you nemonein.  I reported it yesterday as well.  This happened with the move to Plasma 5.22.  I have stopped updates to my other 3 machines in hopes this can be fixed soon.  https://bugs.kde.org/show_bug.cgi?id=438289

**Comment by blue-bullet** [comment]
Updated 2021-06-11 @ 12:47:42 MDT by rob@sigma
I installed autokey-qt today, and it is working again.  I use ctrl+shift+U in autokey to create a line like the one above for use in a comment line.

**Comment by Vinzent03** [comment]
I can confirm, thx for the tip!

**Comment by nemonein** [comment]
It's been fixed by 2.11.6+dfsg-2+20.04+focal+release+build1 (neon repository).
I think it's ok to close this now, so anyone who has authority can close this.

**Comment by josephj11** [comment]
Thanks for following up on this.
